docs/netdata-agent/backup-and-restore-an-agent.md

```diff
@@ -1,5 +1,9 @@
 # Backing up a Netdata Agent
 
+> **Note**
+>
+> With Netdata's decentralised / distributed architecture the data and metrics are always stored locally (on the user's host), it is the user's responsibility to backup, recover and ensure his data is available.
```

Comment:
```suggestion
> [!IMPORTANT] 
> Users are responsible for backing up, recovering, and ensuring their data's availability because Netdata stores data locally on each system due to its decentralized architecture.
```

Reply:
Stress on "Users are responsible" instead of "decentralised / distributed" because that is the point you are making here.

Reply:
@sashwathn see my reasoning for changing the wording above. If you don't like the new version I can undo my changes.